docs/source/notebooks/GP-Circular.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<ul><li>"<strong>As we see, the higher </strong><code><strong>tau</strong></code><strong> is, the less correlated the samples</strong>"</li><li>"<strong>Also, we can extrapolate, getting cycles over and over:</strong>"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4082/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
refactored